### ISSUE TITLE
Register Support and Training Info

### ISSUE THREAD

**Issue by Oguzhanercan** [RobvanGastel/dinov2-finetune#4]
Hi thanks for your work. I want to ask that if this repo supports the register added version of dinov2. Also I want to ask that I am planning to do a training with 112x112 images, do you have any experiments with low resolution images for DinoV2 training?

And last question, in your experiments, how much time does it take to finetune the model (On which gpu also) ?

Thanks.

**Comment by sdracula** [comment]
You can just resize your imgs and labels in data transformation part. And finetuning on single 4080, it takes me several hours to train one epoch.

**Comment by Oguzhanercan** [comment]
Thank you

**Comment by Alighieri1231** [comment]
Hello, is there any reason why using batch size of 8? I am using a bigger batch size (256 with a 4090) but I don't know if in fine-tuning you need to be careful with this. Thanks

**Comment by RobvanGastel** [comment]
Good point, when changing the batch size you should probably linearly scale the learning rate down as well. I dont exactly recall my settings back then.
